index.d.ts

```diff
@@ -13,6 +13,19 @@ declare namespace camelcaseKeys {
 		@default []
 		*/
 		readonly exclude?: ReadonlyArray<string | RegExp>;
+
+		/**
+		Exclude childs of specified object paths(strings in dot notation, for example `prop1.prop2.prop`) from being camelCased.
+
+		@default []
+
+		@example
+		```
+		camelcaseKeys({a_b: 1, a_c: {c_d: 1, c_e: {e_f: 1}}}, {deep: true, stopPaths: ['a_c.c_e']}),
+		//=> {aB: 1, aC: {cD: 1, cE: {e_f: 1}}}
```

Comment:
Can you put all the elements here on separate lines. It's hard to read the example.